src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-count/jj-count.component.ts

```diff
@@ -74,19 +70,25 @@ export class JJCountComponent implements OnInit {
         jjDisputedCountRoP: this.formBuilder.group({
           finding:  [{ value: null, disabled: !this.jjDisputedCount }],
           lesserDescription:  [{ value: null, disabled: !this.jjDisputedCount }],
+          ssProbationCheckbox: [{value: false, disabled: !this.jjDisputedCount }],
           ssProbationDuration:  [{ value: null, disabled: !this.jjDisputedCount }],
           ssProbationConditions:  [{ value: null, disabled: !this.jjDisputedCount }],
+          jailCheckbox: [{value: false, disabled: !this.jjDisputedCount}],
           jailDuration:  [{ value: null, disabled: !this.jjDisputedCount }],
           jailIntermittent:  [{ value: null, disabled: !this.jjDisputedCount }],
+          probationCheckbox: [{value: false, disabled: !this.jjDisputedCount }],
           probationDuration:  [{ value: null, disabled: !this.jjDisputedCount }],
           probationConditions:  [{ value: null, disabled: !this.jjDisputedCount }],
+          drivingProhibitionCheckbox: [{value: false, disabled: !this.jjDisputedCount}],
           drivingProhibition:  [{ value: null, disabled: !this.jjDisputedCount }],
           drivingProhibitionMVASection:  [{ value: null, disabled: !this.jjDisputedCount }],
           dismissed:  [{ value: null, disabled: !this.jjDisputedCount }],
           forWantOfProsecution:  [{ value: null, disabled: !this.jjDisputedCount }],
           withdrawn:  [{ value: null, disabled: !this.jjDisputedCount }],
           abatement:  [{ value: null, disabled: !this.jjDisputedCount }],
+          stayOfProceedingsByCheckbox: [{value: false, disabled: !this.jjDisputedCount}],
           stayOfProceedingsBy:  [{ value: null, disabled: !this.jjDisputedCount }],
+          otherCheckbox: [{value: false, disabled: !this.jjDisputedCount}],
           other:  [{ value: null, disabled: !this.jjDisputedCount }],
```

Comment:
if the whole form will be disabled when there is no jjDisputedCount, it suggested using `!this.jjDisputedCount ?? this.jjDisputedCountRoP.disable()`

Reply:
I attempted this but the look and feel of the disabled count form was significantly and unpleasantly different.  it was worth a try.